UnitTests/GTMSessionFetcherFetchingTest.h

```diff
@@ -119,6 +120,20 @@ extern NSString *const kGTMGettysburgFileName;
 - (instancetype)initWithFailureMessage:(NSString *)failureMessage NS_DESIGNATED_INITIALIZER;
 @end
 
+#pragma mark - User Agent Caching testing helper
+
+typedef NSString *_Nonnull (^UserAgentBlock)(void);
+
+@interface TestUserAgentBlockProvider : NSObject <GTMUserAgentProvider>
+
+@property(atomic, copy) NSString *cachedUserAgent;
+@property(atomic) UserAgentBlock userAgentBlock;
```

Comment:
should probably mark the @property as copy (I know that it is by default for a block, but it makes it clear)

Reply:
done.